src/curated-corpus/components/ExistingProspectCard/ExistingProspectCard.tsx

```diff
@@ -85,29 +100,40 @@ export const ExistingProspectCard: React.FC<ExistingProspectCardProps> = (
           >
             <span>{item.publisher}</span>
           </Typography>
-          <Box py={1}>
-            <Chip
-              variant="outlined"
-              color="primary"
-              label={getDisplayTopic(item.topic)}
-              icon={<LabelOutlinedIcon />}
-            />{' '}
-            {item.isSyndicated && (
+          <Grid container spacing={1}>
+            <Grid item>
               <Chip
                 variant="outlined"
                 color="primary"
```

Comment:
I think some more tweaks are needed here. The markup with the `ProspectCard` component does not appear to be in sync - visually at least. I can see in the video walkthrough that there's less vertical spacing around the tags on the existing prospect card compared to the other card just below it.

I'd also make sure all elements within the card line up to the left - right now the new button doesn't. I'd also capitalise the copy on the new button (we capitalise all other buttons in the app) and change it to to "View recently scheduled runs".